Rlog/View/WorkSpace/WorkSpaceCreateView.swift

```diff
@@ -7,9 +7,111 @@
 
 import SwiftUI
 
+
 struct WorkSpaceCreateView: View {
+    @ObservedObject var viewModel = WorkSpaceCreateViewModel()
+    @State var writingState: WritingState = .hourlyWage
```

Comment:
`writingState` 프로퍼티가 View에서 쓰이는 곳이 없는 것 같아요! 삭제되어야 할 코드인 것 같습니다:)

```suggestion
    @ObservedObject var viewModel = WorkSpaceCreateViewModel()
```

Reply:
뷰모델에 있는걸 까먹고 안지웠었군요 ㅠㅠ 감사합니다!!